src/code/z_player_lib.c

```diff
@@ -1507,6 +1475,8 @@ void Player_PostLimbDrawGameplay(PlayState* play, s32 limbIndex, Gfx** dList, Ve
         if (this->actor.scale.y >= 0.0f) {
             if (!Player_HoldsHookshot(this) && ((hookedActor = this->heldActor) != NULL)) {
                 if (this->stateFlags1 & PLAYER_STATE1_9) {
+                    static Vec3f D_80126128 = { 398.0f, 1419.0f, 244.0f };
+
```

Comment:
This data is needed here for regalloc, and to preserve ordering of data, all the data below needs to be moved in-function